src/__tests__/components/Navbar.test.tsx

```diff
@@ -3,23 +3,59 @@ import '../../__mocks__/matchMediaMock'
 import { Navbar as HospitalRunNavbar } from '@hospitalrun/components'
 import { mount } from 'enzyme'
 import { createMemoryHistory } from 'history'
+import _ from 'lodash'
```

Comment:
prefer the import style of import `{ cloneDeep } from 'loadash/cloneDeep'` that way the whole library doesn't get imported here.

Reply:
✅ in 860fb8c